T4MVCExtensions/AutoNamedRouteAttribute.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web.Mvc.Routing;
+
+namespace System.Web.Mvc
+{
+    [AttributeUsage( AttributeTargets.Method | AttributeTargets.Class, Inherited = false, AllowMultiple = true )]
```

Comment:
The project uses the default VS styling, which does not put spaces inside parenthesis. Can you reformat accordingly? Same in some other files.